### ISSUE TITLE
In IE and window.onerror, stack trace is uninformative

### ISSUE THREAD

**Issue by JoeEven** [stacktracejs/stacktrace.js#9]
Issue 1:

 Run this: 

<pre><code>&lt;html&gt;&lt;head&gt;&lt;script src="stacktrace.js"&gt;&lt;/script&gt;
&lt;script&gt;
function foo() {
    bar(2);
}
function bar(n) {
    if (n &lt; 2)
        abc();
    bar(n-1);
}
    var lastError;
    try {
        // error producing code
        foo();
    } catch(e) {
       lastError = e;
       // do something else with error
    }

    // Returns stacktrace from lastError!
    alert (printStackTrace({e: lastError}).join('\n'));
&lt;/script&gt;
&lt;/head&gt;
&lt;body&gt;&lt;/body&gt;
&lt;/html&gt;
</code></pre> 

<p>Result:</p>

<p>In IE: </p>

<p>{anonymous}([object Error])</p>

<p>printStackTrace(#object)</p>

<p>In FF:
<p>bar(1)@file:///D:/test/javascriptStackTrace/testcase1.html:12</p>

<p>bar(2)@file:///D:/test/javascriptStackTrace/testcase1.html:13</p>

<p>foo()@file:///D:/test/javascriptStackTrace/testcase1.html:8</p>

<p>@file:///D:/test/javascriptStackTrace/testcase1.html:18</p>

<p>What print in FF is expected while the information in IE is a little simple, but I'm not sure whether I'm using it correctly?</p> 

Issue 2:

Run this:

<pre><code>&lt;html&gt;&lt;head&gt;&lt;script src="stacktrace.js"&gt;&lt;/script&gt;
&lt;script&gt;
window.onerror = function(msg, file, line) {
    alert(printStackTrace().join('\n'));
    return true;
}

function foo() {
    bar(2);
}
function bar(n) {
    if (n &lt; 2)
        abc();
    bar(n-1);
}
foo();
&lt;/script&gt;
&lt;/head&gt;
&lt;body&gt;&lt;/body&gt;
&lt;/html&gt;
</code></pre>

<p>Result:</p>

<p>In IE (The same as the first case):</p>

<p>{anonymous}([object Error])</p>

<p>printStackTrace(#object)</p>

<p>In FF (The stack trace is different from the first case):</p>

{anonymous}()@file:///D:/test/javascriptStackTrace/stacktrace.js:74</p>

<p>{anonymous}(null)@file:///D:/test/javascriptStackTrace/stacktrace.js:72</p>

<p>printStackTrace()@file:///D:/test/javascriptStackTrace/stacktrace.js:58</p>

<p>{anonymous}("abc is not defined","file:///D:/test/javascriptStackTrace/testcase1.html",17)@file:///D:/test/javascriptStackTrace/testcase1.html:8</p>

<p>
In short:</p>

<p>
I have two issues, one is the stack trace from IE is different and less information than FF.</p>

<p>
Another one is the stack trace become not so helpful if I print stack in window.onerror.</p>

<p>
Maybe I have some incorrect method in using your library, or they are already known issue?</p>

<p>
Hope for help.</p>

<p>
Thanks</p>
